portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -44,6 +47,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
      */
     String commentsCount = request.getParameter("commentsCount");
 
+    /**
+     * The desired output language code in ISO-639-1 format.
+     */
+    String languageCode = request.getParameter("languageCode");
```

Comment:
Please reflect this change in functionality in the doGet javadoc.

Reply:
Added